www/src/components/account/billing/useBankCard.tsx

```diff
@@ -84,13 +125,109 @@ function useBankCard(setEdit: Dispatch<SetStateAction<boolean>>, address, noCanc
     setLoading(false)
   }, [card, deleteCard, refetch])
 
+  const renderBillingForm = useCallback(() => (
```

Comment:
this pattern is very bug prone and causes gigantic run-on functions.  If we could split it into an entirely separate function component or another file, that would probably be ideal for readability